nextcord/application_command.py

```diff
@@ -995,10 +995,26 @@ def _handle_resolved_message(self, message_data: dict) -> Message:
                 self._state._messages.append(message)
         return message
 
-    def _handle_resolved_user(self, user_data: dict) -> User:
-        """Takes the raw user data payload from Discord and adds it to the state cache."""
-        # This is what I prefer _handle_resolved_message to look like.
-        return self._state.store_user(user_data)
+    def _handle_resolved_user(self, resolved_payload: dict, guild: Optional[Union[Guild, int]] = None) -> Union[User, Member]:
+        """Takes the raw user data payload from Discord and adds it to the state cache.""" # needs changing?
+        user_id, user_payload = list(resolved_payload["users"].items())[0]
+        if not guild:
+            return self._state.store_user(user_payload)
+            
+        else:
```

Comment:
you have an if that returns above, which is essentially a guard clause. That means there's no need to else this section which would make it easier to read.